decidim-meetings/app/views/decidim/meetings/registration_mailer/confirmation.html.erb

```diff
@@ -0,0 +1,3 @@
+<p><%= t(".confirmed", title: translated_attribute(@meeting.title)) %></p>
```

Comment:
Could we add a link tot he meeting with the title?

Reply:
Sure